pkgs/servers/nextcloud/packages/25.json

```diff
@@ -80,19 +90,19 @@
     ]
   },
   "mail": {
-    "sha256": "161ksx7g32fkpwxq3vij2vw6sxblv7xrxggsxnx2wj14ik3wxclv",
-    "url": "https://github.com/nextcloud-releases/mail/releases/download/v2.3.0-alpha.4/mail-v2.3.0-alpha.4.tar.gz",
-    "version": "2.3.0-alpha.4",
+    "sha256": "0sk19y9iwnbw6ci4hq9xxrn5qgnlgqlb5hpzc50i706s1z5n9cmc",
+    "url": "https://github.com/nextcloud-releases/mail/releases/download/v2.2.4/mail-v2.2.4.tar.gz",
```

Comment:
this is a downgrade?

Reply:
before it was an alpha release and I assume it got pulled from the store. Otherwise the updater script needs to be double checked.

Reply:
I haven't looked at how the update script works but 2.2.4 was released after 2.3.0-alpha.4: https://github.com/nextcloud-releases/mail/releases